client/packages.go

```diff
@@ -51,36 +51,36 @@ const (
 	TypeOS        = "os"
 )
 
-// Packages returns the list of packages the system can handle
-func (client *Client) Packages() (map[string]*Package, error) {
-	const errPrefix = "cannot list packages"
+// Snaps returns the list of snaps the system can handle
```

Comment:
Suggestion:
```
// Snaps returns a list of snaps suitable for this system.
```

This removes the constraining sense of the original sentence, which is what @mvo5 was worried about as I understand it.